templates/indicators/collected_data_table.html

```diff
@@ -8,77 +8,80 @@
     }
 </style>
 <script type="text/javascript">
-    $(document).ready(function() {
+    $(document).ready(function () {
         $('#collectedDataTable').DataTable();
     });
-    $("#id_indicator_data_span_count_" + `{{indicator.id}}`).text("{{ indicator.collecteddata_set.count|default:0 }}" );
+    $("#id_indicator_data_span_count_" + `{{indicator.id}}`).text("{{ indicator.collecteddata_set.count|default:0 }}");
 </script>
 {% load assign %}
-{% if indicator.target_frequency  %}
+{% if indicator.target_frequency %}
 
-    <table class="table table-bordered" id="collectedDataTable">
+    <table class="table" id="collectedDataTable">
         <tr>
             <th style="vertical-align:bottom; min-width:180px;">Target Period</th>
             <th style="vertical-align:bottom; text-align: right;">Target</th>
             <th style="vertical-align:bottom; min-width: 125px;">Date Collected</th>
-            <th style="vertical-align:bottom; text-align: right;"">Actual</th>
-            <th style="vertical-align:bottom; text-align: right;"" width="65">% Met</th>
+            <th style="vertical-align:bottom; text-align: right;">Actual</th>
+            <th style="vertical-align:bottom; text-align: right;" width="65">% Met</th>
```

Comment:
Please move these stylings to an external css file (`app.css`)

Reply:
@odenypeter I've made the changes.